packages/manager/apps/hub/src/dashboard/routing.js

```diff
@@ -43,6 +30,19 @@ export default /* @ngInject */ ($stateProvider, $urlRouterProvider) => {
           ),
         );
       },
+      catalogItems: /* @ngInject */ ($http) => {
+        return $http
+          .get('/hub/catalog', {
+            serviceType: 'aapi',
+          })
+          .then((data) => {
+            const { catalog } = data.data.data;
+            return groupBy(
+              filter(catalog.data, ({ highlight }) => highlight),
+              'universe',
+            );
+          });
+      },
```

Comment:
**nitpick:** This results in over-fetching since we will be fetching catalog items for every entry to the dashboard and is not used when the user has ordered at least 1 product.

Reply:
You're right. I can move the call to `/hub/catalog` in the `catalog-items` component to prevent over-fetching.